BookingsApi/BookingsApi/nswag.json

```diff
@@ -116,7 +116,7 @@
       "dateTimeType": "System.DateTimeOffset",
       "timeType": "System.TimeSpan",
       "timeSpanType": "System.TimeSpan",
-      "arrayType": "System.Collections.Generic.ICollection",
+      "arrayType": "System.Collections.Generic.List",
```

Comment:
Is changing the object type in nswag going to break anything for any of the clients consuming it?

Reply:
It shouldn't. out endpoints are returning List in the swagger description and the client is saying ICollection. List is a collection but I'm trying to make sure the client matches how we describe our APIs